vcl_templates/www.vcl.erb

```diff
@@ -354,7 +375,11 @@ sub vcl_deliver {
   if (req.http.Cookie !~ "ABTest-EducationNavigation" || req.url ~ "[\?\&]ABTest-EducationNavigation=" || req.url ~ "^/education(\/|\?|$)") {
     add resp.http.Set-Cookie = "ABTest-EducationNavigation=" req.http.GOVUK-ABTest-EducationNavigation "; expires=" now + 365d "; path=/";
   }
-
+  <% ['SearchMatchLength'].each do |test_name| %>
+  if (req.http.Set-Cookie !~ "ABTest-<%= test_name %>" || req.url ~ "[\?\&]ABTest-<%= test_name %>=") {
+    add resp.http.Set-Cookie = "ABTest-<%= test_name %>=" req.http.GOVUK-ABTest-<%= test_name %> "; expires=" now + 1d "; path=/";
```

Comment:
Yes, the education navigation test has a long expiry time so that users see a consistent version of the site (always A or always B). But it means that there's a lag when we change the B percentage, because some users are stuck in A.

We set the search A/B test expiry to 1 day because thought it would be OK for users to see slightly different search results from day to day, and it gives us more flexibility to change the B percentage. It'd be good to know what Tara and Mark think about that, if we haven't checked with them already?